multiqc/modules/samtools/rmdup.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Samtools rmdup """
+
+import logging
+import re
+from collections import OrderedDict, defaultdict
+from multiqc import config, plots
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class RmdupReportMixin():
+
+    def parse_samtools_rmdup(self):
+        """ Find Samtools rmdup logs and parse their data """
+
+        self.samtools_rmdup = dict()
+        for f in self.find_log_files(config.sp['samtools']['rmdup']):
+            parsed_data = parse_single_report(f['f'])
+            if len(parsed_data) > 0:
```

Comment:
You initialise an empty dictionary in the parsing function, which is always returned. So `len(parsed_data)` is always `1` even if not log files are found currently.

Doesn't really matter if you're going to delete this chunk of code anyway..